huaweicloud/services/apig/resource_huaweicloud_apig_api.go

```diff
@@ -1175,43 +1238,44 @@ func resourceApiCreate(ctx context.Context, d *schema.ResourceData, meta interfa
 	return resourceApiRead(ctx, d, meta)
 }
 
-func analyseBackendParameterValue(origin, value string) string {
+func analyseBackendParameterValue(origin, value string) (paramType, paramValue string) {
 	if origin == string(ParameterTypeSystem) {
 		// The system parameters include gateway parameters and front-end parameters.
-		if ok, err := regexp.MatchString(`\$context\.authorizer\.frontend\.`, value); ok && err == nil {
-			regex, err := regexp.Compile(`\$context\.authorizer\.frontend\.(.*)`)
-			if err != nil {
-				log.Printf("[ERROR] The format of the system parameter value is invalid, want "+
-					"'$context.authorizer.frontend.xxx', but '%s'.", value)
-				return ""
-			}
+		if ok, err := regexp.MatchString(`\$context\.authorizer\.frontend|backend\.`, value); ok && err == nil {
+			regex := regexp.MustCompile(`\$context\.authorizer\.(.*)\.(.*)`)
```

Comment:
```suggestion
                regex := regexp.MustCompile(`\$context\.authorizer\.(frontend|backend)\.(.*)`)
		if regex.MatchString(value) {	
```
you can merge two regxp